app/Http/Controllers/LinkController.php

```diff
@@ -90,4 +88,15 @@ public function performRedirect(Request $request, $short_url, $secret_key=false)
 
         return redirect()->to($long_url);
     }
+    
+    private function handleShortUrlNotExist() {
+    	$urlNotExistHandleType = env('SETTING_REDIRECT_URL_NOT_EXIST');
+    	$urlRedirect = env('SETTING_INDEX_REDIRECT');
```

Comment:
Could we use underscores for these variables to be consistent with other code? e.g `$url_not_found_handle_type`?